### DISCUSSION TITLE
Release of heldout dataset

### DISCUSSION THREAD

**Discussion by sonalkum** [fwillett/speechBCI#18] [General]
Hi, first of all thank you for a great challenge.

I wanted to confirm, if you are planning to release the held out dataset so that we can still continue to test our models, or if you are planning to keep it ongoing on the eval.ai platform?

**Comment by fwillett** [comment]
Hi, glad you enjoyed it! We're planning to keep the benchmark open indefinitely to support further research in the area. No release of the held-out data planned for now.

**Reply by sonalkum** [reply]
So, do you plan to keep the challenge active on the eval.ai platform, so that we can test our models on held-out dataset?

**Reply by fwillett** [reply]
Yes, the eval.ai challenge will be kept active, I changed the end date to 2099, so it should be accepting submissions again.

**Reply by sonalkum** [reply]
Thank you, that will be really helpful for the community. 

Just to bring it to your notice - On eval.ai platform, it is still not accepting the submissions as the dates under "Select Phase" are still old dates, hence it is not letting us select the corresponding radio button. It would be really helpful if you could look into it.

**Reply by fwillett** [reply]
Oops, thank you for pointing that out!! Just fixed it, can you confirm it's working for you now?

**Reply by sonalkum** [reply]
Thanks, it is working now.
